spirv_reflect.h

```diff
@@ -451,11 +525,25 @@ typedef struct SpvReflectEntryPoint {
     uint32_t                        x;
     uint32_t                        y;
     uint32_t                        z;
+    int                             flags; // 0 if just fixed local size
```

Comment:
This addition will change offset of later members and structure size. Will be an abi change.